### ISSUE TITLE
Manage system update - schedule reboot at night no longer working

### ISSUE THREAD

**Issue by neverchained** [eworm-de/routeros-scripts#42]
When running ```/system/script/run packages-update;``` I can no longer schedule a reboot over night.

Output of Terminal:

```
/system/script/run packages-update;
info: Package file routeros already exists.
info: Running backup script backup-cloud before update.
  status: finished

info: Rebooting for update.
Reboot, yes? [y/N]: 
N
action cancelled
```
RouterOS Version 7.9

**Comment by eworm-de** [comment]
Yes, this is a known issue with RouterOS 7.9 and later. The detection whether or not a script is run from terminal is broken. I've opened an issue with Mikrotik, but that is not yet resolved.

Actually I have opened several issues to enhance the situation even more... But non of this has been implemented yet.

Do you have an account in MikroTik support portal? I could share the issues with you if interested...

**Comment by neverchained** [comment]
> Do you have an account in MikroTik support portal? I could share the issues with you if interested...

If Support Portal = https://help.mikrotik.com/ then yes, please share the issue! Thank you!

**Comment by eworm-de** [comment]
What is your account's mail address?

**Comment by neverchained** [comment]
I just emailed you the address.

**Comment by eworm-de** [comment]
Looks like the change did not make it into RouterOS 7.10rc1... Let's hope we will see it before 7.10 final release.

**Comment by eworm-de** [comment]
The fix is in 7.11beta2...

**Comment by eworm-de** [comment]
Finally available in stable release `7.11`.
